packages/wallet/src/redux/utils.ts

```diff
@@ -1,7 +1,9 @@
 import { SideEffects } from './outbox/state';
 
 // Constructs a type that must include all properties of T apart from 'type' and 'stage'
```

Comment:
// Constructs a type that must include all properties of T apart from 'type', 'stage', and 'player'